topics:
  input:
    image: "/anafi/image"
  output:
    boxes: "/yolo/search/boxes"
    detectionImage: "/yolo/search/detectionImage"

publish_detected_image: True

settings:
  class_names: ["human", "floating-objects", "safe_landing", "dummy", "dummy", "dummy"]
  AT_ids: [0, 1, 2, 3, 4, 5]
  detection_confidence: 0.81


